plugins/repository-azure/src/test/java/org/elasticsearch/repositories/azure/AzureBlobStoreRepositoryTests.java

```diff
@@ -64,7 +64,7 @@ protected Settings repositorySettings() {
 
     @Override
     protected Map<String, HttpHandler> createHttpHandlers() {
-        return Collections.singletonMap("/container", new AzureBlobStoreHttpHandler("container"));
+        return Collections.singletonMap("/", new AzureBlobStoreHttpHandler("container"));
```

Comment:
Batch request does not prefix with the container name, right?

Reply:
Jup exactly, that's why I had to expand the context here